platforms/windows/src/node.rs

```diff
@@ -645,9 +645,7 @@ impl IRawElementProviderSimple_Impl for PlatformNode_Impl {
                 }
                 match property_id {
                     UIA_FrameworkIdPropertyId => result = state.toolkit_name().into(),
-                    UIA_ProviderDescriptionPropertyId => {
-                        result = app_and_toolkit_description(state).into()
-                    }
+                    UIA_ProviderDescriptionPropertyId => result = toolkit_description(state).into(),
```

Comment:
In fact, we should remove this property entirely. According to MSDN `UIA_ProviderDescription` is read-only and I can't see the value on Accessibility Insights.

Reply:
I can see the `ProviderDescription` value in NVDA using the NVDA+F1 command. That property does include the toolkit version, unlike `FrameworkId`. But I'm OK with removing it if you think that's better.

Reply:
I am sure I've read on the documentation that this property wasn't meant to be set by providers but I can't find this back. No reason to drop it then.